CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.10.0
+* aAdds `::max-connections-total` and `::max-connections-per-route` to allow the Apache HTTP client values to be overridden.  They default to 20 and 2 (to correlate to the Apahce client behavior) respectively.
```

Comment:
Typos in `aAdds` and `Apahce`. I also think we should call out the dropping of Java 8 support, behavioral changes with the upgraded apache deps and bump the major version. (I also dislike shipping less than 1.0 so now's a great opportunity to fix that.  (: )